### PR TITLE
Document supported environment variables

### PR DESCRIPTION
**Pull request by gurobokum**
## Context

Needs to improve documentation pointing supported environment variables for configuring Studio

### CODE REVIEW

File: docs/env.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Environment variables
+For being able to setup Studio there are a few environment variables
+
+## Backend
+
```
Comment (mjasion) [inline docs/env.md:5]: What do you think of naming this as  `### General` section?

File: docs/env.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+#### BITBUCKET_URL
+#### BITBUCKET_WEBHOOK_URL
+
+## Frontend
+
+#### API_URL (required)
+#### SCM_PROVIDERS (required)
+default=github,gitlab,bitbucket
+
+#### GITHUB_APP_NAME
```
Comment (mjasion) [inline docs/env.md:84]: If in backend is `H3` paragraph  for `Github` Gitlab etc, I would also do it in the frontend. WDYT? 

Maybe here should be `Repositories` or something more gerenal

**Review by mjasion — APPROVED** [review]

File: docs/env.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Environment variables
```
Comment (jesper7) [inline docs/env.md:1]: I think it's better to use a table for each section, i.e.

## GitLab

|Name|Default value|Description|
|---|---|---|
|GITLAB_CLIENT_ID|   |   |
|GITLAB_SECRET_KEY|   |   |
|GITLAB_URL|https://gitlab.com|   |
|GITLAB_WEBHOOK_URL|   |   |
|GITLAB_WEBHOOK_SECRET|   |   |

**Review by jesper7 — CHANGES_REQUESTED** [review]
Great initiative!

**Comment by mvshmakov** [comment]
@JIoJIaJIu could you please add `PLAUSIBLE_DATA_DOMAIN` here as well? It was introduced pretty recently: https://github.com/iterative/studio/pull/4257.

**Comment by mjasion** [comment]
We are closing this PR because we don't want to support studio-selfhosted. We will be migrating to helm-charts soon.

**Pull request closed without merging**